compiler/modules/types/mapping.bal

```diff
@@ -9,6 +9,16 @@ public type MappingAtomicType readonly & record {|
     SemType rest;
 |};
 
+public function mappingAtomicTypeMemberAt(MappingAtomicType mat, string key, int i) returns SemType {
```

Comment:
Get rid of `i` argument (?!)

Don't call the argument `key` since that's a keyword (use `k`).